mcp/include/command_struct.h

```diff
@@ -294,6 +294,64 @@ typedef struct {
   int update_pbob;
 } lj_pbob_t;
 
+typedef struct {
```

Comment:
I would prefer all variable names use a consistent style, either snake_case of camelCase. This struct is also the place to comment each member with important info about what it is, units, etc., so they should all be commented helpfully.

Reply:
This is a cheesy version of the structures defined in star_camera_structs.h (which I will make sure is commented better) and should only be used as a pass through for commands. Normally I'd agree about the variable names but I found that appending only the update_ portion made it easier to read for me, so I'm slightly resistant to change on that front.

Reply:
Sustained!